File: rafael/controle-de-fluxo/EscolheRoupa.cs
class EscolheRoupa 
{
	static void Main() 
	{
		System.Random gerador = new System.Random();
	
		double valor = gerador.NextDouble();

		if (valor < 0.5) 
		{
			System.Console.WriteLine("camiseta preta");
		} 
		else 
		{
			System.Console.WriteLine("camiseta vermelha");
		}

		valor = gerador.NextDouble();

		if (valor < 0.5) 
		{
			System.Console.WriteLine("calça jeans");
		} 
		else 
		{
			System.Console.WriteLine("bermuda");
		}

		valor = gerador.NextDouble();

		if (valor < 0.5) 
		{
			System.Console.WriteLine("tênis");
		} 
		else 
		{
			System.Console.WriteLine("sapato");
		}

		valor = gerador.NextDouble();

		if (valor < 0.5) 
		{
			System.Console.WriteLine("boné");
		} 
		else 
		{
			System.Console.WriteLine("óculos");
		}
	}
}

File: rafael/controle-de-fluxo/EscolheRoupa.java
class EscolheRoupa {
	public static void main(String[] args) {
		double valor = Math.random();

		if (valor < 0.5) {
			System.out.println("camiseta preta");
		} else {
			System.out.println("camiseta vermelha");
		}

		valor = Math.random();

		if (valor < 0.5) {
			System.out.println("calça jeans");
		} else {
			System.out.println("bermuda");
		}

		valor = Math.random();

		if (valor < 0.5) {
			System.out.println("tênis");
		} else {
			System.out.println("sapato");
		}

		valor = Math.random();

		if (valor < 0.5) {
			System.out.println("boné");
		} else {
			System.out.println("óculos");
		}
	}
}
